CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.70.5
+
+### Fixed
+
+- Force dependencies resolution to minor series for 0.70 ([59407a4d34](https://github.com/facebook/react-native/commit/59407a4d3442dddb7f4c29049bf56bbc345f950f) by [@cortinico](https://github.com/cortinico))
+
```

Comment:
I noticed that we are missing the entries for 0.70.4, though

Reply:
for some reason it hasn't been merged yet... https://github.com/facebook/react-native/pull/35081 🤷‍♂️

maybe just add that one here too, and let's close off the other PR and get this one merged